shell.middle/src/main/java/org/metaborg/spoofax/shell/commands/StyledText.java

```diff
@@ -0,0 +1,114 @@
+package org.metaborg.spoofax.shell.commands;
```

Comment:
StyledText is part of a CommandResult, maybe even some sort of placehorder right now.